src/main/webapp/manage-availability.js

```diff
@@ -43,35 +52,49 @@ function addTimeSlotHelper(window) {
     });
 }
 
+// Checks if the tutor tried to add a timeslot with a time in the past, returns true if they did
+function checkMinTime(year, month, day, hour, minute, sessionDate, startTime, window) {
+    // Add 0's when month and day are less than 10 so it matches HTML value format
+    if (month < 10) {
+        month = "0" + month;
+    }
+    if (day < 10) {
+        day = "0" + day;
+    }
+    var yearMonthDay = year + '-' + month + '-' + day;
+    var startTimeArr = startTime.split(':');
+
+    // Add check to see if user-inputted time is in the past
+    if (yearMonthDay == sessionDate) {
+        if (Number(startTimeArr[0]*60) + Number(startTimeArr[1]) < hour*60 + minute) {
```

Comment:
Why is this number? Isn't it better to parse it into a long or an int?

Reply:
I changed it to parseInt